plugins/plugin-core-support/src/test/core-support/tab-navigation.ts

```diff
@@ -16,17 +16,17 @@
 
 import * as common from '@kui-shell/core/tests/lib/common'
 import * as ui from '@kui-shell/core/tests/lib/ui'
-import { theme } from '@kui-shell/core/core/settings'
 const { cli, selectors, keys } = ui
 
 import { tabButtonSelector } from '@kui-shell/plugin-core-support/lib/new-tab'
+const navigatable = '.kui--tab-navigatable'
 
-describe('tab navigation', function(this: common.ISuite) {
+describe('tab keyboard navigation', function(this: common.ISuite) {
```

Comment:
If we do `TEST_FILTER='tab navigation'  npm test`, and the test fails, we will also get afterHook error from `common.pDescribe('core new tab from pty active tab via click')`. I haven't figure out why, but adding a `keyboard` to the describe seems to be a temporary workaround.